### PR TITLE
Docs: Translating.rst reworked and line-shifted

### PR DESCRIPTION
**Pull request by comradekingu**
Can I put https://github.com/WeblateOrg/weblate/issues/6169#issuecomment-860242734 here?

### CODE REVIEW

File: docs/user/translating.rst [hunk 1]
```diff
@@ -36,8 +36,8 @@ Translation links
 
 Having navigated to a component, a set of links lead to its actual translation.
 The translation is further divided into individual checks, like
-:guilabel:`Not translated strings` or :guilabel:`Strings needing action`. If the whole project
-is translated, without error, :guilabel:`All strings` is still available.
+:guilabel:`Not translated strings` or :guilabel:`Strings needing action`.
+If the whole project is translated, without error, :guilabel:`All strings` is still available.
```
Comment (comradekingu) [inline docs/user/translating.rst:40]:
```suggestion
If the whole project is translated without any errors, :guilabel:`All strings` is still available.
```
This actually meant the label is always available when a project is translated, as opposed to some of the time.

File: docs/user/translating.rst [hunk 2]
```diff
@@ -49,9 +49,9 @@ Suggestions
 
     Actual permissions might vary depending on your Weblate configuration.
 
-Anonymous users can only (by default) forward suggestions. Doing so is still
-available to signed-in users, in cases where uncertainty about the translation
-arises, prompting other translators to review it.
+Anonymous users can only (by default) forward suggestions.
+Doing so is still available to signed-in users, in cases where uncertainty
```
Comment (comradekingu) [inline docs/user/translating.rst:53]:
```suggestion
Doing so is still available for signed-in users, in cases where uncertainty
```

**Comment by nijel** [comment]
> Can I put [#6169 (comment)](https://github.com/WeblateOrg/weblate/issues/6169#issuecomment-860242734) here?

Please no, the bigger the pull request is, the harder it is to review.

File: docs/user/translating.rst [hunk 3]
```diff
@@ -1,21 +1,18 @@
 Translating using Weblate
 =========================
 
-Thank you for interest in translating using Weblate. Projects can either be
-set up for direct translation, or by way of accepting suggestions made by
-users without accounts.
+Projects either accept translations directly,
+or when a number of votes is reached.
```
Comment (orangesunny) [inline docs/user/translating.rst:5]: Suggestions can be used with voting, but it is not the only use case, thus it can’t be described like this.

**Review by orangesunny — APPROVED** [review]
good to go now

**Comment by orangesunny** [comment]
@nijel, please work out the failing checks

**Pull request merged**